### PR TITLE
Commiting Chapter Five

### PR DESCRIPTION
**Pull request by MarkWignall**

### CODE REVIEW

File: chapter05/mdw/Ex-2.pl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/perl
+
+use feature "say";
+
+my @words;
+
+while(<>){
```
Comment (thatguy59) [inline chapter05/mdw/Ex-2.pl:7]: I can't remember if there is an actual difference or not, but `push @words <>` should get rid of the need for the while loop without any change to how it works, I think, maybe, possibly.....

Comment (qianliok) [inline chapter05/mdw/Ex-2.pl:7]: or `chomp( my @words= <STDIN>)` will work as well

File: chapter05/mdw/Ex-2.pl [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+push @words, $_;
+
+}
+
+say "123456789012345678901234567890";
```
Comment (qianliok) [inline chapter05/mdw/Ex-2.pl:13]: `"1234567890"x3` can save a bit of typing. :smiley:

File: chapter05/mdw/Ex-1.pl [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/perl
+
```
Comment (codavex) [inline chapter05/mdw/Ex-1.pl:2]: No strict and no warnings? Who knows what monstrosities lie below ...

**Pull request merged**